tools/travis/before_install.sh

```diff
@@ -98,11 +98,9 @@ echo "<settings>
 
 echo -e "\nInstalling some python packages: "
 
-pip install --user --upgrade pip
```

Comment:
So we are not upgrading pip, do we need to document which version is expected?

Reply:
The default one that is v9.0 is okay.